### DISCUSSION TITLE
目前Issues里有一堆标着`等待反馈者`的Issue

### DISCUSSION THREAD

**Discussion by xxtg666** [Meloong-Git/PCL#4131] [讨论]
目前Issues里有一堆标着`等待反馈者`的Issue，估计是很多人也不用GitHub也不看邮件，导致那些回复他们自己都看不到
PCL2有没有考虑加一个提醒功能？可以在第一次从PCL2中打开反馈页面时提示输入GitHub用户名（之后可在设置里修改），然后每次打开PCL2的时候在后台发个网络请求检测一下有没有这个用户名的标着`等待反馈者`的Issue，有的话可以弹窗提示跳转

**Comment by zkitefly** [comment]
github api 连通性不行吧

**Reply by Pigeon0v0** [reply]
不行另说

**Reply by Deep-Dark-Forest** [reply]
Github又不像google之类的完全墙，至少还有一些人能够直连，有了总比没有好一点

**Comment by LTCatt** [comment]
目前够用 =。=

**Comment by SqrtTan** [comment]
之前全是等待是因为上学去了，现在真的不能理解（

**Reply by kong-que-dnf123** [reply]
事实上
我们昨天刚会考完
明天开始期末考试。。。

**Reply by WForst-Breeze** [reply]
其实也有人还在上课啊（悲）

**Reply by SqrtTan** [reply]
可怜（虽然我这次考差了
